docs/packages.rst

```diff
@@ -171,17 +171,14 @@ Packages
     docs_dimod/sdk_index
     docs_binarycsp/sdk_index
     docs_cloud/sdk_index
-    docs_gate/sdk_index
```

Comment:
I believe you meant to remove the next line, not this one!

Reply:
oops. Thanks!